File: MODUL-3/SOAL 1/PRAK301-2410817120017-RIKAFAULIANARAHMI.py
a, b, c = map(int, input("Masukkan 3 angka (dipisahkan spasi): ").split())

if a > b:
  a, b = b, a

if a > c:
  a, c = c, a

if b > c:
  b, c = c, b

print("Angka terurut:", a, b, c)

File: MODUL-3/SOAL 1/PRAK301-2410817120017-RIKAFAULIANARAHMI.c
#include <stdio.h>

int main() {
  int a, b, c;

  printf("Masukkan 3 angka (dipisahkan spasi): ");
  scanf("%d %d %d", &a, &b, &c);

  if (a > b) {
    int temp = a;
    a = b;
    b = temp;
  }

  if (a > c) {
    int temp = a;
    a = c;
    c = temp;
  }

  if (b > c) {
    int temp = b;
    b = c;
    c = temp;
  }

  printf("Angka terurut: %d %d %d\n", a, b, c);

  return 0;
}